### DISCUSSION TITLE
购买了专业版几个星期不能用，请退款

### DISCUSSION THREAD

**Discussion by Lewis0000221** [getlantern/lantern-forum-cn#906] [General]
购买了专业版几个星期不能用，请退款
连接后一直上不了外网，谷歌页面无法显示
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@Lewis0000221 您好. 您要不先别退款? 重新下载最新版本是看. 
如果真没办法, 那您就提供我支付单据和账号邮箱. 我帮您提交申请. 但如果可以的话, 下载最新版先, 别急着退款. 谢谢!

**Reply by Lewis0000221** [reply]
@Mike777Khoo 你好，下了最新版版本还是上不了，昨天试了一整天都是跟之前一样上不了，手机总是不能连接，请安排退款。本人也很想支持蓝灯，但无奈真的手机用不了，希望能理解并退款，谢谢！
账号邮箱：andylewis@163.com
![微信图片_20240907140828](https://github.com/user-attachments/assets/b8330fea-d49a-4760-85bc-bdf2dea65fb1)
